### DISCUSSION TITLE
update harian, perlu atau tidak?

### DISCUSSION THREAD

**Discussion by malikshi** [helmiau/HelmiWrt-OS#5] [Q&A] [answered]
Bang mau nanya nih, kan maintener nya sering commit sama rilisin versi baru perlu ga sih harus upgrade sedangkan yang dicommit bukan major updates/issues

**Comment by helmiau** [comment, marked as answer]
> Bang mau nanya nih, kan maintener nya sering commit sama rilisin versi baru perlu ga sih harus upgrade sedangkan yang dicommit bukan major updates/issues

perlu upgrade atau tidaknya kembali ke masing-masing pengguna aja bang. kalo sekiranya pengen update ya di update, kalo engga ya tinggal pake yang ada sekarang

**Reply by Siplek** [reply]
Mas mohon maaf ijin bertanya,kalau mau update comit2 yg dimaksud ts atas bagaimana ya ? Apa harus download lagi yg sysupgrade trus flash frimware by luci,

**Reply by helmiau** [reply]
> Mas mohon maaf ijin bertanya,kalau mau update comit2 yg dimaksud ts atas bagaimana ya ? Apa harus download lagi yg sysupgrade trus flash frimware by luci,

harus download ulang, burn ulang fw yg terbaru

**Reply by Siplek** [reply]
Siap terima kasih infonya🙏🏿🙏🏿

**Reply by malikshi** [reply]
Kalau butuh yg tiap update ga ribet flash lagi, sya ada reference firmware nya, dan cuman tools kek passwall, ssr+ harus edit custum sendiri krn ga ada file ipk nya, kalau butuhnya openclash,libernet xderm bisa install manual karena ada ipk file nya. Sesuai kebutuhan, sya pakai punya repo dari wulfy, dan mas helmiau

**Reply by helmiau** [reply]
> Kalau butuh yg tiap update ga ribet flash lagi, sya ada reference firmware nya, dan cuman tools kek passwall, ssr+ harus edit custum sendiri krn ga ada file ipk nya, kalau butuhnya openclash,libernet xderm bisa install manual karena ada ipk file nya. Sesuai kebutuhan, sya pakai punya repo dari wulfy, dan mas helmiau

sysupgradenya ane belom bisa buat update OTA kayaknya (blm pernah coba)
